### PR TITLE
CEDS-2889 Set up the Exit Survey for Declaration Service

### PR DESCRIPTION
**Pull request by janeturner**
Add link to Exit Survey on /customs-declare-exports/declaration/confirmation page and on the user log out page.

### CODE REVIEW

File: app/views/components/feedback_frontend.scala.html [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+@import views.html.components.gds._
+@import config.AppConfig
+
+@this(paragraphBody: paragraphBody,
+        link: link,
+        appConfig: AppConfig)
+
+@()(implicit messages: Messages)
+
+<div>
```
Comment (russellthorn) [inline app/views/components/feedback_frontend.scala.html:26]: very small comment, but does this page need this div? It doesn't seem to be doing anything, especially without a class.

Comment (janeturner) [reply app/views/components/feedback_frontend.scala.html]: Good point, I copied this from the submission page into the component and didn't change it. Removed it now.

File: app/config/AppConfig.scala [hunk 2]
```diff
@@ -53,6 +53,7 @@ class AppConfig @Inject()(
   lazy val relevantLicensesUrl = loadConfig("urls.relevantLicenses")
   lazy val serviceAvailabilityUrl = loadConfig("urls.serviceAvailability")
   lazy val customsMovementsFrontendUrl = loadConfig("urls.customsMovementsFrontend")
+  lazy val feedbackFrontendUrl = loadConfig("urls.feedbackFrontendUrl")
```
Comment (glidester) [inline app/config/AppConfig.scala:56]: Using the name 'feedback' here for the exist survey I think is a bit confusing as it clashes with the 'giveFeedbackLink' below (line 66, used in the phaseBanner component). Could we call this new one 'exitSurvey' to more clearly distinguish it from the header feedback link?

**Review by glidester — CHANGES_REQUESTED** [review]

File: app/config/AppConfig.scala (hunk 2 above)
Comment (PauloPortugal) [reply app/config/AppConfig.scala]: @glidester bike-shedding big time, but he is actually right! It is just a shame that he did not give the same suggestion on my SFUS work :stuck_out_tongue:

**Review by glidester — APPROVED** [review]
LGTM :slightly_smiling_face:

**Pull request merged**